<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>War3EditorTool</class>
 <widget class="QWidget" name="War3EditorTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_ability">
      <attribute name="title">
       <string> 技能</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>721</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_cost">
          <property name="text">
           <string>魔法消耗</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_cooldown">
          <property name="text">
           <string>冷却时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_range">
          <property name="text">
           <string>施法距离</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_duration">
          <property name="text">
           <string>持续时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_area">
          <property name="text">
           <string>影响范围</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>721</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_abilId">
          <property name="text">
           <string>A001</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Main Description:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="textEdit_mainDesc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ability ID:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Max Level:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_maxLv">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>生成:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_res">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>721</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>460</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>预览:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_preview">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>721</width>
         <height>181</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_copyToClipboard">
       <property name="geometry">
        <rect>
         <x>614</x>
         <y>420</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>复制到剪切板</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>其他</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
